pom.xml

```diff
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>io.github.thebusybiscuit</groupId>
     <artifactId>CSCoreLib2</artifactId>
-    <version>0.4.2</version>
+    <version>0.4.3</version>
```

Comment:
I think this should already be 0.5

Reply:
With 0.5 I meant actual 0.5, not 0.4.5 XD

Reply:
REEEE